test/unit/tokens/ERC20RebaseDistributor.t.sol

```diff
@@ -484,4 +484,162 @@ contract ERC20RebaseDistributorUnitTest is Test {
             assertApproxEqAbs(token.nonRebasingSupply(), nonRebasingSupplyBefore, maxError);
         }
     }
+
+    function testCanExitRebaseAfterEnteringRebase(uint256 distributionAmount) public {
+        vm.assume(distributionAmount > 0);
+        vm.assume(distributionAmount < 1_000_000_000e18);
```

Comment:
This is a rather broad check (the overwhelming amount of probability is for a `uint256` to exceed 10^27), so it would be more efficient to use `bound` from forge-std (or just modulo `distributionAmount` with `10**27`) rather than `vm.assume` here (and in the other tests using as well):

https://book.getfoundry.sh/cheatcodes/assume